files-community/Files
# Fix: Fixed issue where the System version of VS Code wasn't detected

**Resolved / Related Issues**

**Validation**
How did you test these changes?
   1. Install VS Code (System)
   2. Navigate to a git repository
